glide.yaml

```diff
@@ -20,3 +20,5 @@ import:
 - package: github.com/arschles/kubeapp
   subpackages:
   - api
+- package: github.com/docker/docker
+  version: 2b27fe17a1b3fb8472fde96d768fa70996adf201
```

Comment:
@krancour an update - without the version set, a `glide up` seems to work as well, from a non-existent `vendor` dir:

```console
ENG000656:workflow-manager aaronschlesinger$ glide up
[INFO] Downloading dependencies. Please wait...
[INFO] Fetching updates for github.com/gorilla/mux.
[INFO] Fetching updates for github.com/arschles/testsrv.
[INFO] Fetching updates for k8s.io/kubernetes.
[INFO] Fetching updates for github.com/arschles/assert.
[INFO] Fetching updates for github.com/kelseyhightower/envconfig.
[INFO] Fetching updates for github.com/satori/go.uuid.
[INFO] Fetching updates for speter.net/go/exp/math/dec/inf.
[INFO] Fetching updates for github.com/arschles/kubeapp.
[INFO] Fetching updates for github.com/go-swagger/go-swagger.
[INFO] Fetching updates for github.com/docker/docker.
[INFO] Setting version for github.com/go-swagger/go-swagger to 0.5.0.
[INFO] Detected semantic version. Setting version for k8s.io/kubernetes to v1.2.5-beta.0.
[INFO] Resolving imports
[INFO] Fetching github.com/gorilla/context into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Found Godeps.json file.
[INFO] Fetching github.com/google/cadvisor/info/v1 into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/google/cadvisor to 546a3771589bdb356777c646c6eca24914fdd48b.
[INFO] Fetching github.com/davecgh/go-spew/spew into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/davecgh/go-spew to 3e6e67c4dcea3ac2f25fd4731abc0e1deaf36216.
[INFO] Fetching github.com/ugorji/go/codec into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/ugorji/go to f4485b318aadd133842532f841dc205a8e339d74.
[INFO] Fetching golang.org/x/net/context into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for golang.org/x/net to c2528b2dd8352441850638a8bb678c2ad056fd3e.
[INFO] Setting version for speter.net/go/exp/math/dec/inf to 42ca6cd68aa922bc3f32f1e056e61b65945d9ad7.
[INFO] Fetching github.com/golang/glog into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/golang/glog to 44145f04b68cf362d9c4df2182967c2275eaefed.
[INFO] Found GB manifest file.
[INFO] Fetching github.com/asaskevich/govalidator into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/asaskevich/govalidator to 9699ab6b38bee2e02cd3fe8b99ecf67665395c96.
[INFO] Fetching gopkg.in/yaml.v2 into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for gopkg.in/yaml.v2 to d466437aa4adc35830964cffc5b5f262c63ddcb4.
[INFO] Found Godeps.json file.
[INFO] Fetching github.com/google/gofuzz into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/google/gofuzz to bbcb9da2d746f8bdbd6a936686a0a6067ada0ec5.
[INFO] Fetching github.com/emicklei/go-restful/swagger into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/emicklei/go-restful to 777bb3f19bcafe2575ffb2a3e46af92509ae9594.
[INFO] Fetching github.com/blang/semver into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/blang/semver to 31b736133b98f26d5e078ec9eb591666edfd091f.
[INFO] Fetching github.com/prometheus/client_golang/prometheus into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/prometheus/client_golang to 3b78d7a77f51ccbc364d4bc170920153022cfd08.
[INFO] Fetching github.com/spf13/pflag into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/spf13/pflag to 08b1a584251b5b62f458943640fc8ebd4d50aaa5.
[INFO] Fetching github.com/juju/ratelimit into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/juju/ratelimit to 77ed1c8a01217656d2080ad51981f6e99adaa177.
[INFO] Fetching github.com/opencontainers/runc/libcontainer/cgroups/fs into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/opencontainers/runc to 7ca2aa4873aea7cb4265b1726acb24b90d8726c6.
[INFO] Fetching github.com/pborman/uuid into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/pborman/uuid to ca53cad383cad2479bbba7f7a1a05797ec1386e4.
[INFO] Fetching github.com/beorn7/perks/quantile into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/beorn7/perks to b965b613227fddccbfffe13eae360ed3fa822f8d.
[INFO] Fetching github.com/golang/protobuf/proto into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/golang/protobuf to b982704f8bb716bb608144408cff30e15fbde841.
[INFO] Fetching github.com/prometheus/client_model/go into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/prometheus/client_model to fa8ad6fec33561be4280a8f0514318c79d7f6cb6.
[INFO] Fetching github.com/prometheus/common/expfmt into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/prometheus/common to ef7a9a5fb138aa5d3a19988537606226869a0390.
[INFO] Fetching github.com/prometheus/procfs into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/prometheus/procfs to 490cc6eb5fa45bf8a8b7b73c8bc82a8160e8531d.
[INFO] Fetching github.com/ghodss/yaml into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/ghodss/yaml to 73d445a93680fa1a78ae23a5839bad48f32ba1ee.
[INFO] Found Godeps.json file.
[INFO] Setting version for github.com/docker/docker to 0f5c9d301b9b1cca66b3ea0f9dec3b5317d3686d.
[INFO] Fetching bitbucket.org/ww/goautoneg into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for bitbucket.org/ww/goautoneg to '75cd24fc2f2c2a2088577d12123ddee5f54e0675'.
[INFO] Fetching github.com/matttproud/golang_protobuf_extensions/pbutil into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/matttproud/golang_protobuf_extensions to fc2b8d3a73c4867e51861bbdd5ae3c1f0869dd6a.
[INFO] Fetching github.com/docker/go-units into /Users/aaronschlesinger/gocode/src/github.com/deis/workflow-manager/vendor
[INFO] Setting version for github.com/docker/go-units to 0bbddae09c5a5419a8c6dcdd7ff90da3d450393b.
[INFO] Downloading dependencies. Please wait...
[INFO] Setting references for remaining imports
[INFO] Project relies on 35 dependencies.
```